APIReplayDownloader/database_load.py

```diff
@@ -1,16 +1,74 @@
 import database
-from playerdata import get_leaderboard_by_page
+import transaction
+from playerdata import get_user_by_id
 from time import sleep
 
+leaderboard_4k_db = database.open_db("db/leaderboard_4k.fs")
+leaderboard_4k_connection = database.get_connection(leaderboard_4k_db)
 
-lb_db = database.open_db("db/leaderboard.fs")
-connection = database.get_connection(db)
+player_db = database.open_db("db/players.fs")
+player_connection = database.get_connection(player_db)
 
-for i in range(0, 71):
-    lb_page = get_leaderboard_by_page(i)
-    for user in lb_page:
-        print("Adding lb entry to database: ", database.insert_leaderboard_entry(connection, user))
-    sleep(1)
+print("Adding players from 4k leaderboard to database...")
 
-connection.close()
-lb_db.close()
+num = 0
+for i in range(1, 10050):  # for all ranks in the 4k leaderboard db
+    player_id = database.get_player_id_by_rank(leaderboard_4k_connection, i)
+
+    if player_id != -1:
+        if database.get_player(player_connection, player_id) != {}:
+            print(f"Player (id {player_id}) already in database. Skipping...")
+            continue
+        player_profile = get_user_by_id(player_id, 1)  # 1 means 4k
+        if player_profile:
+            transaction.begin()
+            print(f"Adding {player_profile['info']['username']} (id {player_id}) to database:",
+                  database.insert_player(player_connection, player_profile))
+            transaction.commit()
```

Comment:
ideally we should have many inserts per transaction so that we dont run into file size explosion due to too many consecutive commits. Something like 25 inserts per transaction.